subprojects/docs/src/snippets/files/fileCollections/kotlin/build.gradle.kts

```diff
@@ -53,25 +55,28 @@ tasks.register("usage") {
         val file: File = collection.singleFile
 
         // Add and subtract collections
-        val union = collection + layout.files("src/file2.txt")
-        val difference = collection - layout.files("src/file2.txt")
+        val union = collection + projectLayout.files("src/file2.txt")
+        val difference = collection - projectLayout.files("src/file2.txt")
 
         // end::usage[]
     }
 }
 
 tasks.register("filterTextFiles") {
+    // Copy collection property to a local variable for configuration cache support.
+    val collection: FileCollection = collection
+    val projectDirectory: Directory = layout.projectDirectory
```

Comment:
🔴 Prefer to omit the type declaration in Kotlin unless it's necessary for correctness.

Reply:
Done 👍